plugins/package-managers/spm/src/main/kotlin/Spm.kt

```diff
@@ -141,13 +141,13 @@ class Spm(
 
     private fun projectFromDefinitionFile(definitionFile: File): Project {
         val vcsInfo = VersionControlSystem.forDirectory(definitionFile.parentFile)?.getInfo().orEmpty()
-        val (author, project) = parseAuthorAndProjectFromRepo(repositoryURL = vcsInfo.url)
+        val (author, _) = parseAuthorAndProjectFromRepo(repositoryURL = vcsInfo.url)
 
         val projectIdentifier = Identifier(
             type = managerName,
             version = vcsInfo.revision,
             namespace = author.orEmpty(),
-            name = project ?: definitionFile.parentFile.relativeTo(analysisRoot).invariantSeparatorsPath
+            name = definitionFile.parentFile.relativeTo(analysisRoot).invariantSeparatorsPath
```

Comment:
IMO we should strive to use the `getFallbackProjectName()` helper function like @mnonnenmacher recently did as part of https://github.com/oss-review-toolkit/ort/pull/8117.